flink-connector/src/main/java/com/datastrato/gravitino/flink/connector/utils/TypeUtils.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.flink.connector.utils;
+
+import com.datastrato.gravitino.rel.types.Type;
+import com.datastrato.gravitino.rel.types.Types;
+import org.apache.flink.table.api.DataTypes;
+import org.apache.flink.table.types.DataType;
+import org.apache.flink.table.types.logical.LogicalType;
+
+public class TypeUtils {
```

Comment:
could you add UT for this class?

Reply:
@FANNG1 This class is simple. Do we need to add UT for it?

Reply:
I think we need UT because this class is important for flink connector